### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by gestchild**
Added SVG icons from Code and Theory and edited to allow styling with CSS, when making use of SVG sprite and  'use' element

### CODE REVIEW

File: client/images/icons/actions/Bookmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 22"><title>Artboard 1 copy 52</title><path class="icon-sprite__shape" fill="#1d1d1d" d="M18,1H4A3,3,0,0,0,1,4V19a2,2,0,0,0,2,2,2,2,0,0,0,1-.29L11,16l7,4.71A2,2,0,0,0,19,21a2,2,0,0,0,2-2V4A3,3,0,0,0,18,1ZM12.12,14.34a2,2,0,0,0-2.24,0L3,19V4A1,1,0,0,1,4,3H18a1,1,0,0,1,1,1V19Z"/></svg>
```
Comment (davidpmccormick) [inline client/images/icons/actions/Bookmark.svg]: I imagine more often than not we'll want to (aria-) hide icons from screenreaders, but in the cases we don't would the title be read? If so, we should probably make the `<title>`s meaningful where possible?

Comment (gestchild) [inline client/images/icons/actions/Bookmark.svg]: The relevant title will actually be a child of the containing svg (not the one on the icon itself, which will be in the shadow DOM, if that makes sense).

I've just pushed a component for icons, which allows you to pass a variable to determine if the icon is decorative or not. If it is, then no title is added (and aria-hidden is added to the SVG). If it isn't decorative, you can pass in a title to the component.

Comment (davidpmccormick) [inline client/images/icons/actions/Bookmark.svg]: Ah cool – does that mean we can remove the `<title>`s altogether?

Comment (gestchild) [inline client/images/icons/actions/Bookmark.svg]: Good point, yes

**Pull request merged**